edx_repo_tools/drip_survey/upload.py

```diff
@@ -0,0 +1,59 @@
+"""
+
+"""
+import csv
+from datetime import date
+import hashlib
+import sys
+
+import click
+from edx_repo_tools.data import pass_repo_tools_data
+
+NAME = 'FirstName'
+EMAIL = 'PrimaryEmail'
+WEEK = 'Week'
+ASSOCIATED_WITH = 'AssociatedWith'
+UNSUBSCRIBED = 'Unsubscribed'
+
+def associated_with(person):
```

Comment:
Yeah. I thought you had a more in depth set of things you were extracting, though, so I kept this definition local, for now. Happy to move it if you think there's enough there.

I think I'm ok with occasional contractors (like OpenCraft) acting as internal, rather than edX.